crates/cli/src/subcmd_craft_deploy/meta.rs

```diff
@@ -0,0 +1,77 @@
+use serde::{Deserialize, Serialize};
+use serde_json::Value as Json;
+
+use svm_layout::{FixedLayoutBuilder, Id};
+use svm_types::{CtorsSection, DataSection};
+
+// Note: at the time of writing (2021-07-26), we don't care about most fields
+// within the "meta" JSON. As such, the [`TemplateMeta`] sub-entities can be
+// removed or replaced with [`serde_json::Value`]. However, let's leave them for
+// future use once we want to add support for optional template sections.
+
+/// A fully parsed JSON from the "meta" output of the SVM SDK.
+#[derive(Debug, Serialize, Deserialize)]
+pub struct TemplateMeta {
+    schema: Vec<TemplateMetaSchema>,
+    api: Vec<TemplateMetaApi>,
+}
+
+impl TemplateMeta {
+    pub fn ctors_section(&self) -> CtorsSection {
+        let ctors = self
+            .api
+            .iter()
+            .filter(|api_entry| api_entry.is_ctor)
```

Comment:
can you rename `api_entry` to `export`? 
Probably using the term `Exports` instead of `API` under `svm-codec` would be more precise.